demo-simple/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,10 @@
 <resources>
   <string name="app_name">Lite Download Manager</string>
+
+  <string-array name="file_sizes">
+    <item>10MB</item>
+    <item>20MB</item>
+    <item>50MB</item>
+    <item>100MB</item>
```

Comment:
We can only go up to 100MB in file sizes, GH limits prevents larger files without additional customisation of the repo.

Reply:
Boooooooo @github